src/Cli/dotnet/ReleasePropertyProjectLocator.cs

```diff
@@ -234,9 +243,9 @@ public IEnumerable<string> GetCustomDefaultConfigurationValueIfSpecified()
         /// <param name="project">The project under a solution to evaluate.</param>
         /// <param name="projectFullPath">The full hard-coded path of the project.</param>
         /// <returns>True if the project is not supported by ProjectInstance class or appears to be invalid.</returns>
-        private bool IsUnanalyzableProjectInSolution(SlnProject project, string projectFullPath)
+        private bool IsUnanalyzableProjectInSolution(SolutionProjectModel project, string projectFullPath)
         {
-            return project.TypeGuid == solutionFolderGuid || project.TypeGuid == sharedProjectGuid || !IsValidProjectFilePath(projectFullPath);
+            return project.TypeId.ToString() == solutionFolderGuid || project.TypeId.ToString() == sharedProjectGuid || !IsValidProjectFilePath(projectFullPath);
```

Comment:
Is TypeId always the same guid as it would've been had you used TypeGuid for a SlnProject?

Reply:
You mean if they're equivalent? In that case, yes